app/src/components/common/modal_connect_wallet/index.tsx

```diff
@@ -155,12 +157,14 @@ export const ModalConnectWallet = (props: Props) => {
     localStorage.setItem('CONNECTOR', walletSelected)
   }
 
+  const doesMetamaskExist = 'ethereum' in window || 'web3' in window
+
   return (
     <>
       {!context.account && (
         <ModalWrapper isOpen={isOpen} onRequestClose={onClickCloseButton} title={`Choose a Wallet`}>
-          <ItemMetamask />
-          <ItemWallet />
+          {doesMetamaskExist && <ItemMetamask />}
```

Comment:
I think it's better to disable it rather than hiding it. @leolower @gabitoesmiapodo wdyt?

Reply:
Yes.

Reply:
I will need to add some opacity, like in this image, because this component is a Div
![Screenshot_20200204_181840](https://user-images.githubusercontent.com/1144028/73787969-fa59e900-477a-11ea-8268-9625fa71a479.png)

Reply:
That's fine.

Reply:
Done!